prime-angular-frontend/src/app/modules/adjudication/shared/components/site-registration-tabs/site-registration-tabs.component.scss

```diff
@@ -0,0 +1,4 @@
+::ng-deep .mat-tab-body-content {
```

Comment:
Can we add a similar comment to `::ng-deep` that we did for the other uses of it in the application.  Feels like we've used it quite a few times, but each time I keep thinking about when it will be dropped due to it being deprecated until the HTML spec for ShadowDOM gets official formalized.